package.json

```diff
@@ -46,9 +47,8 @@
     "moment": "^2.22.2",
     "react": "16.9.0",
     "react-intl": "^3.3.1",
-    "react-native": "0.61.2",
+    "react-native": "^0.61.5",
```

Comment:
please revert to rn version, we should follow this guide for upgrade https://github.com/react-native-community/rn-diff-purge/compare/release/0.61.2..release/0.61.5

Reply:
There is no guide, it is difference between version and I checked changelog, only bugs are fixed, no major changes, that's why I upgraded.

Reply:
yes it is not guide but offical documents recommend to this way. i know it not has any major update but if we not apply all changes we can encounter to some issues on feature updates

Reply:
How to apply all changes? I thought yarn fetches all updates with you add new version. Let me know steps and I will do it.